src/style.css

```diff
@@ -1 +1,68 @@
-/* Styles go here */
+ul li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+body {
+  margin: 0;
+  --linkColor: #00acdc;
+}
+
+.header {
+  margin: 0 50px;
```

Comment:
You don't need to use margin here. It looks like padding for inner content. So, use better:
```
padding-left: 50px;
padding-right: 50px;
```
And as a bonus - your tests will pass.